MaybeGame/EnemyStats.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+
+namespace MaybeGame
+{
+    public class EnemyStats
+    {
+        public int Strength { get; set; }
+        public int Stamina { get; set; }
+        public int Agility { get; set; }
+        public int Dexterity { get; set; }
+        public int Health { get; set; }
+        public int Defence { get; set; }
+        public double DodgeChance { get; set; }
+        public int Speed { get; set; }
+        public int AttackPower { get; set; }
+
+        public int SetStrength(string enemy)
+        {
+            // Strength is the quality or state of being physically strong.
+
+            var strength = 5;
+
+            switch (enemy)
+            {
+                case "Zombie":
+                    strength += 2;
+                    break;
+
+                case "Skeleton":
+                    strength += 2;
+                    break;
+
+                case "Bandit":
+                    strength += 3;
+                    break;
+
+                case "Theif":
+                    strength += 2;
+                    break;
+
+                case "Warrior":
+                    strength += 4;
+                    break;
+                
+                case "Berserker":
+                    strength += 7;
+                    break;
+                
+                case "Rat":
+                    strength += 0;
+                    break;
+
+                default:
+                    Console.WriteLine("Error 404: Race not found");
+                    break;
+            }
+
+            return strength;
+        }
+        public int SetStamina(string enemy)
+        {
+            // Stamina is the ability to sustain prolonged physical or mental effort.
+
+            var stamina = 5;
+
+            switch (enemy)
+            {
+                case "Zombie":
+                    stamina += 2;
```

Comment:
Merge some cases with keyword **when**

```suggestion
case when EnemyType.Zombie or EnemyType.Skeleton
                    stamina += 2;
```